Add question marks

davidpucheta opened the pull request:
Add opening question marks for Spanish translation of document.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

davidpucheta committed:
Add question marks

Add opening question marks for Spanish translation of document

RandellDawson commented:
@davidpucheta The translation for these headers is incorrect.  They should never have been translated to be questions anyway.  Can you please review the [English version](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/README.md) of this document and correct the translation of the headers?

Thank you.

davidpucheta committed:
Udate spanish translation

Change the spanish translation of 'Reporting Bugs and Issues' and 'Reporting Security Issues' headers to be aligned with the original English version.

davidpucheta commented:
@RandellDawson Changed the text to be aligned with the english version

QuincyLarson reviewed (approved):
These improvements look solid. Thanks again, @davidpucheta.
